### ISSUE TITLE
Weird reporting of TrailingSemicolon

### ISSUE THREAD

**Issue by Anahkiasen** [brigade/scss-lint#200]
Using 0.26, here is the error I get:

```
public/app/sass/modules/_data.scss:61 [W] TrailingSemicolon: Declaration should not have a space before the terminating semicolon
```

Here is the code the linter is complaining about (the precise line is the include one):

``` scss
@include breakpoint($medium-range-max) {
    .table {
        display: none;
    }
}
```
